<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgApOne</class>
 <widget class="QDialog" name="DlgApOne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>541</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="2" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="bnStart">
           <property name="text">
            <string>Розпочати набір</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnConcretion">
           <property name="text">
            <string>Згущення</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnRarefaction_1">
           <property name="text">
            <string>Розкачка 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnRarefaction_2">
           <property name="text">
            <string>Розкачка 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnGrowth">
           <property name="text">
            <string>Ріст</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Корекція</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QDial" name="dl_Kor">
           <property name="minimum">
            <number>-25</number>
           </property>
           <property name="maximum">
            <number>25</number>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="notchesVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="sb_Kor">
           <property name="minimum">
            <number>-25</number>
           </property>
           <property name="maximum">
            <number>25</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bnStop">
        <property name="text">
         <string>Зупинити</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="bnSetup">
     <property name="text">
      <string>Параметри</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>127</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="bnExit">
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="apMnemo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>421</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Поточна стадія</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_State"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bnExit</sender>
   <signal>clicked()</signal>
   <receiver>DlgApOne</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
